test/tools/ossfuzz/solProto.proto

```diff
@@ -0,0 +1,108 @@
+/*
+	This file is part of solidity.
+
+	solidity is free software: you can redistribute it and/or modify
+	it under the terms of the GNU General Public License as published by
+	the Free Software Foundation, either version 3 of the License, or
+	(at your option) any later version.
+
+	solidity is distributed in the hope that it will be useful,
+	but WITHOUT ANY WARRANTY; without even the implied warranty of
+	MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+	GNU General Public License for more details.
+
+	You should have received a copy of the GNU General Public License
+	along with solidity.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+syntax = "proto2";
+
+message InterfaceFunction {
+    enum StateMutability {
+        PURE = 0;
+        VIEW = 1;
+        PAYABLE = 2;
```

Comment:
What about `NONPAYABLE`? (nonview but also nonpayable)

Reply:
Good point, I completely missed this.

Reply:
Added nonpayable to spec.

Reply:
I think nonpayable is not a valid state mutability any longer: please see https://github.com/ethereum/solidity/issues/8736

Reply:
There was a misunderstanding here on my part. Nonpayable is indeed a valid state mutability but it is indicated by the absence of a mutability keyword, not presence of one. So, `NONPAYABLE` should be added as suggested by @leonardoalt